### DISCUSSION TITLE
EDDI won't install

### DISCUSSION THREAD

**Discussion by HyperCatsNSpace** [EDCD/EDDI#2375] [EDDI Standalone Q&A] [answered]
When running EDDI-4.0.1.exe, I get an error saying "Windows cannot access the specified device, path, or file. You may not have the appropriate permissions to access the item." I have tried using other versions of EDDI, changing permissions in the file properties, and disabling antivirus and Windows Firewall, but the error persists. Has anyone else had the same issue, and how might I fix it?

**Comment by Tkael** [comment, marked as answer]
I'm afraid that is very unusual. You'll probably find that it installs ok on other computers (if so then it is specifically this one computer that seems to have the issue).

**Reply by Tkael** [reply]
If the file system doesn't think you have ownership of the file then you could perhaps [try this](https://www.howtogeek.com/301768/how-to-take-ownership-of-files-and-folders-in-windows/).

**Reply by HyperCatsNSpace** [reply]
Thanks for the reply! Unfortunately, that doesn't seem to be the issue because the properties menu already states that I am the file owner. I think that the issue might involve Inno Setup because I tried compiling EDDI instead of using the installer, which requires Inno Setup. However, the Inno Setup installer gave me the same error when I tried running it.

**Reply by Tkael** [reply]
That's good to know, and yes that may be having an impact. I assume that you're following the instructions [here](https://github.com/EDCD/EDDI/wiki/Running-from-source) then?

Are you willing to give the pre-compiled installer version a try?

**Reply by HyperCatsNSpace** [reply]
I was able to get it working, my antivirus was bugged and was disabling random programs even when turned off. Thank you for taking the time to reply, I appreciate it!
